web/auth/jwt.go

```diff
@@ -56,39 +55,35 @@ func (tm *TokenManager) JWTUpdateClaims(db database.Database, userID uint64) (*C
 	return newClaims, nil
 }
 
+// TODO(vera): needs access to the database to update changes there
 // Update will remove a user ID at the index from the list
-func (tm *TokenManager) Remove(index uint64) {
+func (tm *TokenManager) Remove(index uint64, db database.Database) {
 	updatedTokenList := append(tm.TokensToUpdate[:index], tm.TokensToUpdate[index+1:]...)
 	tm.TokensToUpdate = updatedTokenList
 }
 
 // Add adds a new UserID to the list of users who require an updated JWT token
-func (tm *TokenManager) Add(userID uint64) {
+func (tm *TokenManager) Add(userID uint64, logger *zap.SugaredLogger, db database.Database) {
```

Comment:
Maybe the `TokenManager` struct should have a copy of the `db` and `logger` so that you don't need to pass these to all methods.

Reply:
Note that I've added a helper in `log/logger.go`, mainly to be used from tests like so:
```go
logger := log.Zap(false)
sugaredLogger := log.Zap(false).Sugar()
```
The `verbose` flag turns logging on/off. For tests we typically only want to enable logging when we get test failures to better understand why there is a problem...